mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1862,8 +1862,12 @@ impl ElectrumClient {
                         let len = CompactInteger::from(headers.count);
                         let mut serialized = serialize(&len).take();
                         serialized.extend(headers.hex.0.into_iter());
-                        let mut reader =
-                            Reader::new_with_coin_params(serialized.as_slice(), CoinVariant::Standard, Some(coin_name));
+                        let coin_variant = if coin_name == "LBC" {
```

Comment:
Could you please add `coin_variant_by_ticker` helper instead and use it everywhere?